cmd/okctl/create.go

```diff
@@ -255,6 +257,11 @@ and database subnets.`,
 				Message: "Are you ready to start?",
 			}
 
+			err = servicequota.CheckQuotas(o.CloudProvider)
```

Comment:
Here we should be newing up the checks, and passing them in to the checker implementation, it is easier to control the setup of the individual checkers, which ones we want to add, etc. See the comments further down the stack 👍

Reply:
Done